core/src/main/java/org/apache/accumulo/core/client/impl/TabletServerBatchWriter.java

```diff
@@ -733,7 +733,7 @@ else if (Tables.getTableState(context.getInstance(), table) == TableState.OFFLIN
     void queueMutations(final MutationSet mutationsToSend) throws InterruptedException {
       if (null == mutationsToSend)
         return;
-      binningThreadPool.execute(new Runnable() {
+      binningThreadPool.execute(Trace.wrap(new Runnable() {
```

Comment:
Looking at `org.apache.htrace.wrappers.TraceRunnable`, it's not really designed to be used as you're suggesting.  One would hope to be able to write:
```java
threadPool.execute(new TraceRunnable() { ... })
```

But the best that `TraceRunnable` allows is:
```java
threadPool.execute(new TraceRunnable(new Runnable() {...}));
```